pkgs/applications/office/wpsoffice/default.nix

```diff
@@ -54,6 +53,10 @@ stdenv.mkDerivation rec {
     nspr
     mesa
     libtiff
+    udev
```

Comment:
Why not put `(lib.getLib udev)` directly into `runtimeDependencies` instead of `patchelf`?